templates/todo/projects/csharp-cosmos-sql/.repo/bicep/infra/main.bicep

```diff
@@ -24,6 +24,9 @@ param logAnalyticsName string = ''
 param resourceGroupName string = ''
 param webServiceName string = ''
 
+@description('Flag to use Azure API Management to mediate the calls between the frontend and the backend API')
```

Comment:
How long does APIM take to deploy?

What is the cost?

Reply:
I recorded 1m16s to deploy the service with the Consumption tier. $0.042 per 10,000 calls

Reply:
Nice!